docs/api.rst

```diff
@@ -139,7 +139,6 @@ Handling user creation, sign-in, log-out and querying
               "settings": {
                   "have_premium": true,
                   "version": "6",
-                  "last_write_ts": 1571552172,
```

Comment:
Where is this going to if it's removed from here?

Reply:
We decided to remove `last_write_ts` from the API because it is not being used on the frontend.

Reply:
Is not that we decided unilaterally. We asked and confirmed that this field is not used